blocks/editable/block.scss

```diff
@@ -0,0 +1,16 @@
+.drop-cap-true .blocks-editable__tinymce:not( :focus )  {
```

Comment:
@youknowriad yes, that was the original plan, a `has-drop-cap` class added to `p`.